native/swift/Tests/wordpress-api/Support/Mocks.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+import WordPressAPI
+
+final class MockAppNotifier: WpAppNotifier {
```

Comment:
This class is not used.

Reply:
I don't have a good method for injecting a secret WP.com token quite yet without making it public, but I made a test class to debug this endpoint with, and I used `MockAppNotifier` there.

So I'm using the test helper, just not in the repo code yet 😅